tests/virt_autotest/virt_utils.pm

```diff
@@ -171,7 +183,7 @@ sub clean_up_red_disks {
     if (($disks_nu_length eq $disks_nu_num) && $disks_nu_num && $fs_type_supported) {
         foreach my $item (@disks_nu_array) {
             if ($item =~ /^\/dev\/sd[b-z]$/) {
-                assert_script_run("wipefs -a -f $item && $make_fs_cmd $item", $wait_script);
+                assert_script_run("echo \"y\\n\" | $wipe_fs_cmd $item &&  echo \"y\\n\" | $make_fs_cmd $item", $wait_script);
```

Comment:
Would you please explain and comment in code why this \"y\\n\" needed?

Reply:
Another layer of protection. Force process to proceed if question encountered.

Reply:
shouldn't the "-f" option be enough? Otherwise I would recommend to use the "yes" command and not echo. Could you maybe change that?